test/Store.test.js

```diff
@@ -227,6 +228,7 @@ describe('Store', function() {
       spy.calledOnce.should.eql(true);
       spy.alwaysCalledWith('', {
```

Comment:
Could you add a test suite that explictly sets up two stores, dispatches an action to each, and ensures that we only dispatch to proper port names?